src/pokemon.c

```diff
@@ -6134,8 +6134,7 @@ struct OakSpeechNidoranFStruct *OakSpeechNidoranFSetup(u8 battlePosition, bool8
             battlePosition = 1;
         if (battlePosition > 8)
             battlePosition = 8;
-        // The following two statements refused to cooperate. 
-        sOakSpeechNidoranResources->spriteCount = battlePosition;
+        sOakSpeechNidoranResources->spriteCount = (battlePosition << 16) >> 16;
```

Comment:
this is a fake match

Reply:
This is not a fake match. An ameteur programmer can write this.

Reply:
This is 100% not what the original programmer wrote.  It's better than no match, though.

Reply:
Its not 100%. It's more like 95%. Once again, an ameteur can write this.